decimal.go

```diff
@@ -385,7 +385,12 @@ func (d Decimal) Cmp(d2 Decimal) int {
 	return rd.value.Cmp(rd2.value)
 }
 
-// Equals returns whether the numbers represented by d and d2 are equal.
+// Equal returns whether the numbers represented by d and d2 are equal.
+func (d Decimal) Equal(d2 Decimal) bool {
+	return d.Cmp(d2) == 0
+}
+
+// Equals is deprecated, please use Equal method instead
 func (d Decimal) Equals(d2 Decimal) bool {
 	return d.Cmp(d2) == 0
```

Comment:
@victorquinn Done. 

I changed Equals to Equal, because I also implemented code in asserts library to detect that structure implements method Equal and to use this method to compare objects instead of calling reflect.DeepEqual. 
there is my PR to asserts library https://github.com/stretchr/testify/pull/394